tests/GiraffeGenerator.IntegrationTests/specs/specWithArguments.yaml

```diff
@@ -63,6 +63,22 @@ paths:
         determined using above GET api.
       operationId: perform-search
       parameters:
+        - name: start
+          description: Starting record number. Default value is 0.
+          in: query
+          schema:
+            type: integer
+            default: 0
+        - name: rows
+          description: >-
+            Specify number of rows to be returned. If you run the search
+            with default values, in the response you will see 'numFound'
+            attribute which will tell the number of records available in
+            the dataset.
+          in: query
+          schema:
+            type: integer
```

Comment:
same question. It should be either 2 different specs or 2 different test cases (with/without parameters/form)

Reply:
There is specWithSchemas for parameterless case.
There is `list-searchable-fields` operation in this spec which have only route and query parameters.
And there is `perform-search` (the operation you've commented on) which defines every kind of parameters supported.
Do you think we need a query-only and route-only cases?

Reply:
Added path-only, query-only, route-only, form-only and json-only test cases